tcs-client/src/test/java/com/transformuk/hee/tis/tcs/client/service/impl/TcsServiceImplTest.java

```diff
@@ -402,4 +402,25 @@ public void getGmcDetailsByIdShouldThrowErrorWhenNotFound() {
     // RestTemplate hasn't been set up to find any GMC, so should throw exception
     testObj.getGmcDetailsById(20L);
   }
+
+  @Test
+  public void patchCurriculumMembershipShouldReturnSavedDto() {
+    CurriculumMembershipDTO dto = new CurriculumMembershipDTO();
+
+    String url = "http://localhost:9999/tcs/api/curriculum-memberships";
+
+    HttpHeaders headers = new HttpHeaders();
+    HttpEntity<CurriculumMembershipDTO> httpEntity = new HttpEntity<>(dto, headers);
+    ResponseEntity responseEntity = new ResponseEntity(dto, HttpStatus.OK);
+    doReturn(responseEntity).when(restTemplate).exchange(url, HttpMethod.PATCH, httpEntity,
+        new ParameterizedTypeReference<CurriculumMembershipDTO>() {
+        });
+
+    CurriculumMembershipDTO result = testObj.patchCurriculumMembership(dto);
+
+    assertThat("Unexpected result", result, is(dto));
```

Comment:
Why are we asserting an "unexpected result"?

Reply:
It was mere copy of some previous tests. My bad.  I have changed the text to "Expected result".

Reply:
I think "Unexpected result" is correct. 
`assertThat` method shows this description when the assertion fails.

Reply:
Yes, that's correct, Yafang. Tested and reverted back.